### PR TITLE
Add newsletter form to docs footer and more…

### PR DESCRIPTION
**Pull request by laurakalbag**
## Layout/design:
- Adds the email newsletter signup to the footer STA-3164
- Makes the copyright HTML more accessible
- Adds a Mastodon link/icon to the footer
- Redesigns the footer to a more minimal responsive design
- Makes sidebar and page menu border lines more subtle in style

Before: 
CleanShot 2023-02-28 at 11 24 28@2x
CleanShot 2023-02-28 at 11 24 15@2x

After:
CleanShot 2023-02-28 at 11 24 38@2x
CleanShot 2023-02-28 at 11 24 45@2x

## Docs: 
- Fixes a typo on the intro to statecharts
- Prevents confusion on parallel states by no longer referring to them as parent states
- Adds links to new templates on Templates page STA-3379

### CODE REVIEW

File: docs/state-machines-and-statecharts.mdx [hunk 1]
```diff
@@ -176,7 +176,7 @@ An atomic state is a state that doesn’t have any child states. **Waiting**, **
 
 ## Parallel states
 
-A parallel state is a parallel state where all of its child states, also known as regions, are active simultaneously. The regions are separated inside the parallel state container by a dashed line.
+A parallel state is a tate where all of its child states, also known as regions, are active simultaneously. The regions are separated inside the parallel state container by a dashed line.
```
Comment (farskid) [inline docs/state-machines-and-statecharts.mdx:179]:
```diff
- A parallel state is a tate
+ A parallel state is a state
```

**Comment by farskid** [comment]
@laurakalbag Is there a reason we don't use the exact sentence for describing parallel states in all those places?

**Comment by laurakalbag** [comment]
> @laurakalbag Is there a reason we don't use the exact sentence for describing parallel states in all those places?

Glossary is least specific, so it links to more specific explanations. The main explanation gives some details more relevant to the visual Studio representation of parallel states. Otherwise I’m trying to keep them the same!

File: docusaurus.config.js [hunk 2]
```diff
@@ -158,6 +161,9 @@ const config = {
...
           },
+          {
+            html: '<form action="registry/api/newsletter/signup" method="post" target="popupwindow" class="embeddable-buttondown-form state" data-active><label for="bd-email">Subscribe to our newsletter</label><div class="fields"><input type="email" name="email" id="bd-email" required="" placeholder="your@email.com"><input type="hidden" value="1" name="embed"><input type="submit" class="white" class="copy-button" value="Subscribe"></div></form>'
```
Comment (mellson) [inline docusaurus.config.js:165]: Let's change the action URL to `https://stately.ai/registry/api/newsletter/signup`. Then sign-up will work from everywhere our docs are deployed.

**Review by mellson — APPROVED** [review]
Pre-approval. When the action URL is updated, this is good to go 👏🏻

File: docusaurus.config.js (hunk 2 above)
Comment (laurakalbag) [reply docusaurus.config.js]: @mellson thank you! Done!

**Pull request merged**